change-notes/1.19/analysis-cpp.md

```diff
@@ -23,8 +23,12 @@
 | Expression has no effect | Fewer false positive results | Expressions in template instantiations are now excluded from this query. |
 | Global could be static | Fewer false positive results | Variables with declarations in header files are now excluded from this query. |
 | Resource not released in destructor | Fewer false positive results | Placement new is now excluded from the query. Also fixed an issue where false positives could occur if the destructor body was not in the snapshot. |
+| Memory is never freed | Fewer false positive results | This query now accounts for C++ _placement new_, which returns a pointer that does not need to be freed. |
 | Missing return statement (`cpp/missing-return`) | Visible by default | The precision of this query has been increased from 'medium' to 'high', which makes it visible by default in LGTM. It was 'medium' in release 1.17 and 1.18 because it had false positives due to an extractor bug that was fixed in 1.18. |
 | Missing return statement | Fewer false positive results | The query is now produces correct results when a function returns a template-dependent type, or makes a non-returning call to another function. |
+| Multiplication result converted to larger type (`cpp/integer-multiplication-cast-to-long`) | Fewer false positive results | Char-typed numbers are no longer considered to potentially large. |
+| Non-virtual destructor in base class (`cpp/virtual-destructor`) | Fewer false positive results | This query was renamed from "No virtual destructor" and moved from file name `AV Rule 78.ql` to `NonVirtualDestructorInBaseClass.ql`. The new version ignores base classes with non-public destructors since we consider those to be adequately protected. |
```

Comment:
I don't like the phrasing "renamed" and "moved" here as it suggests that `AV Rule 78.ql` no longer exists.  I think we should say that it was "copied".

Reply:
I agree. I also find my explanation here to be more convoluted that it ought to be.

Reply:
I've attempted to update the explanation now. I made it more precise but possibly even more convoluted.